### PR TITLE
Implement Issue #2

### PR DESCRIPTION
**Pull request by martinhath**
If this isn't hot, I don't know what is

### CODE REVIEW

**Comment by martinhath** [comment]
@finnss

**Comment by finnss** [comment]
So the 
`controller.on('mention,direct_mention',function(bot,message) {` 
is called twice when you send a message with two mentions? Wouldn't have guessed.

Anyway, damn! Impressive!

File: index.js [hunk 1]
```diff
@@ -64,12 +88,24 @@ controller.on('mention,direct_mention',function(bot,message) {
...
+            const makeMention = user => '<@' + user.id + '>';
+
+            var text = '';
+            if (rekts.length > 1) {
+              // comma separate them nicely :)
+              const last = rekts[numberOfMentions - 1];
+              const rest = rekts.slice(0, numberOfMentions - 1);
+              text += rest.map(makeMention).join(', ');
+              text += ', and ' + makeMention(last);
+            } else {
```
Comment (finnss) [inline index.js:101]: sexy

**Comment by martinhath** [comment]
Well, not really, but with a direct mention `@noen: hey ho lets go`, the mention part of the string is removed from the `text` field, so `message.text == ': hey ho lets go'` (might be without `:` too - I don't remember). However, this is not the case with usual mentions.

**Pull request merged**